templates/vue/src/components/NodeModal.vue

```diff
@@ -10,16 +10,18 @@
     body-class="p-0"
     @hidden="close"
   >
-    <div v-if="formErrors.length" class="modal-header-row">
-      <b-alert id="tapestry-modal-form-errors" variant="danger" show>
-        <ul>
-          <li v-for="error in formErrors" :key="error">{{ error }}</li>
-        </ul>
-      </b-alert>
-    </div>
     <b-container fluid class="px-0" data-qa="node-modal">
       <b-overlay :show="loading" variant="white">
-        <b-tabs card>
+        <div v-if="submissionError || formErrors.length" class="error-wrapper">
+          <h5>Node cannot be saved due to the following error(s):</h5>
+          <ul>
+            <li v-for="error in formErrors" :key="error">{{ error }}</li>
+            <li v-if="submissionError" data-qa="modal-submit-error">
+              {{ submissionError }}
+            </li>
```

Comment:
Can we add the submissionError as a part of formErrors so we avoid having two variables to keep track of errors to display? We can just amend the API error to the formErrors array maybe? You may need to move `setTapestryErrorReporting` functionality to the `handleSubmit()` section (not sure).